dp/utils/io.py

```diff
@@ -7,29 +7,85 @@
 
 
 def read_config(path: str) -> Dict[str, Any]:
+    """
+
+    Args:
+      path: str: 
```

Comment:
Incomplete explanation of the parameter in docstring.

Reply:
Updated.